aws/lambda/node/log-api-key-to-cloudwatch/index.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+const AWS = require('aws-sdk');
+const dataFormatRegex = /(csv|tsv|json|yaml|ttl)$/;
+
+exports.handler = (event, context, callback) => {
+   const request = event.Records[0].cf.request;
+   
+   // Let request continue immediately then carry out our logic
+   callback(null, request);
```

Comment:
Am not _sure_ this actually does anything? As per:
> When the callback is called, the Lambda function exits only after the Node.js event loop is empty (the Node.js event loop is not the same as the event that was passed as a parameter). 

https://docs.aws.amazon.com/lambda/latest/dg/nodejs-prog-model-handler.html

Reply:
I can confirm that we definitely need to call the callback, but agree that it can probably be moved to the end of the function because the function definitely does not terminate at this point.